lib/rucio/client/ruleclient.py

```diff
@@ -227,3 +227,18 @@ def examine_replication_rule(self, rule_id):
             return next(self._load_json_data(r))
         exc_cls, exc_msg = self._get_exception(r.headers, r.status_code)
         raise exc_cls(exc_msg)
+
+    def examine_replication_locks(self, rule_id):
```

Comment:
I would suggest to name this `get_replica_locks` or `list_replica_locks`.
The `examine` from the previous method was a bit specific to finding transfer errors, as the return dictionary is specifically prepared for this. Also please adapt the description text.

Reply:
Done, sorry about that!